release_tools/github.py

```diff
@@ -88,6 +89,25 @@ def download_archive(self, branch, save_to_path, ball="zipball"):
             archive.extractall(save_to_path)
             print "Extracted"
 
+    def download_release_history(self, path):
+        url = "https://api.github.com/repos/{owner}/{repo}/releases{token}"\
+              .format(owner=self.owner, repo=self.repo, token=self.access_token_postfix())
+        response = requests.get(url)
+        if response.status_code == 200:
+            print "Writing to file..."
+            with open(path, 'w') as f:
+                f.write(self._release_history_contents(response.json()))
+            print "done."
+        else:
+            print "Something went wrong, contents cannot be downloaded"
```

Comment:
Alternatively one could raise an error here and only write this message in the `cli.py`, then the module is easier to use as a library.